bin/heralding

```diff
@@ -61,19 +61,20 @@ def setup_logging(logfile, verbose):
     root_logger.addHandler(file_log)
 
 
-def break_if_python_not_supported():
-    if sys.version_info[0:2] < (3, 5):
-        raise Exception("Wrong python version! Your Python interpreter must be 3.5.0 or above!")
-
-
 class LogFilter(logging.Filter):
     def filter(self, rec):
-        if rec.name == 'paramiko.transport':
+        """Disabling internal aiosmtpd logger"""
+        if rec.name == 'mail.log':
             return False
         else:
             return True
 
 
+def break_if_python_not_supported():
+    if sys.version_info[0:2] < (3, 5):
```

Comment:
Actually, shouldn't this be 3.5.2? await/async syntax was implemented with 3.5.2 if I remember correctly...